.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: required
+language: java
+jdk:
+  - oraclejdk8
+services:
+  - docker
+before_install:
+  - sudo apt-get -qq update
```

Comment:
possible more readable alternative for apt-get -qq is apt-get -q=2 (quiet level 2)